addons/l10n_ro_efactura/models/account_move.py

```diff
@@ -250,13 +250,13 @@ def _l10n_ro_edi_fetch_invoice_sending_documents(self):
                     company=invoice.company_id,
                     key_download=result['key_download'],
                     session=session,
+                    status=result['state_status'],
                 )
                 to_delete_documents |= invoice._l10n_ro_edi_get_sending_and_failed_documents()
                 final_result['key_loading'] = active_sending_document.key_loading
                 if 'error' in final_result:
-                    final_result['attachment_raw'] = previous_raw
                     invoice._l10n_ro_edi_create_document_invoice_sending_failed(
-                        message=final_result['error'],
+                        message=final_result['error'].replace('\t', ''),
```

Comment:
Personal preference or asked by the po ? or was the \t displayed in the message ?

Reply:
Some error messages contained \t and would create a lot of unpleasant indentation :3